### ISSUE TITLE
Forget outfile

### ISSUE THREAD

**Issue by Droppix** [LaboratoryB/stegosaurus#1]
Hi,

You have 2 issues in functions : encodeString and encodeFile
You have forget option 'outfile'

``` javascript
exports.encodeString = function(infile,outfile,messagestring,callback) {

    if (typeof callback == 'undefined') {
        callback = function(){};
    }

    var sendoptions = {
        encode: true,
        decode: false,
        target: infile,
        outfile: outfile,
        message: messagestring,
    }

    stegosaurus.handler(sendoptions,function(err){
        callback(err);
    });
}
```

Regards

**Comment by dougbtv** [comment]
Thanks for the heads up Robert, appreciate it. I'll publish a fix this
afternoon.

**Comment by dougbtv** [comment]
Should up and published on NPM @ version `0.0.9` with the fix in place. I'll keep the issue open for a day or two in case you were to find anything else. Thanks again, always appreciate the eyeballs.

**Comment by dougbtv** [comment]
I think I've got it all under control, I realized that how I missed it is that I set a default `out.png` and it would always write `out.png`, and  I had a faulty test that always looks for... you guessed it, `out.png`. 

I went ahead and added a formal unit test and a travis-ci build process which runs the test, and published the changes to NPM. So, we should be good to go. Thanks for opening the issue.
